README.md

```diff
@@ -75,10 +85,12 @@ Create a `secretproviderclasses` resource to provide provider-specific parameter
 
     ```
 
-    | Name           | Required | Description                                                     | Default Value |
-    | -------------- | -------- | --------------------------------------------------------------- | ------------- |
-    | provider       | yes      | specify name of the provider                                    | ""            |
-    | usePodIdentity | no       | specify access mode: service principal or pod identity          | "false"       |
+    | Name                   | Required | Description                                                     | Default Value |
+    | --------------         | -------- | --------------------------------------------------------------- | ------------- |
+    | provider               | yes      | specify name of the provider                                    | ""            |
+    | usePodIdentity         | no       | specify access mode: service principal or pod identity          | "false"       |
+    | useVMManagedIdentity   | OPTIONAL available for version > 0.0.4       | specify access mode: use a service principal or pod identity or vm managed identity    |  "false"|
+    | userAssignedIdentityID | OPTIONAL available for version > 0.0.4       | The user assigned identity ID is required for VMSS User Assigned Managed Identity mode  | ""       |
```

Comment:
same

Reply:
I see in your comment below `leave empty to use system assigned managed identity` can we make this more obvious here?